server/flaskr/blueprints/meeting_agenda.py

```
@@ -67,6 +74,41 @@ def get_meeting_agenda(id: str):
+@meeting_agendas_bp.post("/<int:id>/reviews")
+@input_middleware(
+    LambdaBuilder(
+        ("objective", IntValidator(1,5)),
+        ("smoothRunning", IntValidator(1,5)),
+        ("preparation", IntValidator(1,5)),
+        ("length", IntValidator(1,5)),
+        ("respect", IntValidator(1,5)),
+        ("comments", StringValidator()),
+    )
+)
+def create_meeting_review(objective, smoothRunning, preparation, length, respect, comments, id: int):
```

Ruff (0.15.18)

[error] Function argument `id` is shadowing a Python builtin

(A002)